src/create_confirmation_request.cc

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "create_confirmation_request.h"
+#include "ads_serve_helper.h"
+#include "security_helper.h"
+
+#include "base/logging.h"
+#include "base/json/json_writer.h"
+#include "base/values.h"
+
+namespace confirmations {
+
+CreateConfirmationRequest::CreateConfirmationRequest() = default;
```

Comment:
convention here would be `ConfirmationRequestBuilder` or just `ConfirmationRequest`

Reply:
CreateConfirmationRequest is due to the server API naming the task CreateConfirmation as the API creates a confirmation

Reply:
But I do agree these classes should have Builder, i,e, CreateConfirmationRequestBuilder